src/templates/InfoTable.tsx

```diff
@@ -1,79 +1,138 @@
+                                            left:
+                                                index === 1
+                                                    ? stickyParam.width
+                                                    : '',
+                                        }}>
+                                        <div
+                                            style={{
+                                                display: 'flex',
+                                                justifyContent: 'space-between',
+                                            }}>
+                                            <Typography>
+                                                {title.label}
+                                            </Typography>
+                                            {title.value ? (
+                                                <TableSortButton
+                                                    TableSort={tableSort}
+                                                    setTableSort={setTableSort}
+                                                    nameSort={title.value}
+                                                />
+                                            ) : (
+                                                ''
+                                            )}
+                                        </div>
+                                    </th>
+                                );
+                            })}
                         </tr>
                     </thead>
-                    <tbody>
-                        {viewData.map((item: Items) => {
-                            return (
-                                <tr>
-                                    <td className="sticky">
-                                        <Typography>{item.title}</Typography>
-                                    </td>
-                                    <td
-                                        className="sticky"
-                                        style={{ left: stickyParam.width }}>
-                                        <Typography>{item.author}</Typography>
-                                    </td>
-                                    <td>
-                                        <Typography>{item.keywords}</Typography>
-                                    </td>
-                                    <td>
-                                        <Typography>
-                                            {item.summary.content.replace(
-                                                '&amp',
-                                                '&'
-                                            )}
-                                        </Typography>
-                                    </td>
-                                </tr>
-                            );
-                        })}
-                    </tbody>
+                    {isLoading ? (
```

Comment:
good practice is not to choose between the loader and the data, but to display loader over the data with some overlay layout. try it

Reply:
Corrected.